runtime-builder/build.sh

```diff
@@ -26,7 +26,7 @@ usage() { echo "Usage: $0 <project> <go_version>"; exit 1; }
 debian_digest()
 {
   # We sometimes have issue with 'describe', use the --log-http flag to get more info if it fails.
-  local digest="$(gcloud container images describe gcr.io/google-appengine/debian8:latest | \
+  local digest="$(gcloud alpha container images describe gcr.io/google-appengine/debian8:latest | \
```

Comment:
I don't think there is a need to record the debian digest in the user's image.  

(1) It's implementation detail of the builder and that may change.
(2) A debian image can be used by different versions of the builder.

What perhaps you may want to record in the user's image is the builder's unique tag, i.e. use BUILD_TAG env below.